### PR TITLE
Add Rubinius 3.17–3.19

### PR DESCRIPTION
**Pull request by deepj**

### CODE REVIEW

File: share/ruby-build/rbx-3.19 [hunk 1]
```diff
@@ -0,0 +1,4 @@
+require_llvm 3.5
+install_package "openssl-1.0.2f" "https://www.openssl.org/source/openssl-1.0.2f.tar.gz#932b4ee4def2b434f85435d9e3e19ca8ba99ce9a065a61524b429a9d5e9b2e9c" mac_openssl --if has_broken_mac_openssl
+install_package "rubinius-3.19
```
Comment (hsbt) [inline share/ruby-build/rbx-3.19]: Please fix this line.

Comment (deepj) [inline share/ruby-build/rbx-3.19]: @hsbt Thanks, done.

**Comment by hsbt** [comment]
Thanks always.

**Comment by deepj** [comment]
@hsbt thanks :)

**Pull request merged**